wiki/tfchain/tfchain_home.md

```diff
@@ -0,0 +1,62 @@
+# TF Chain
+
+TF Chain is the base layer for everything that interacts with the grid. Nodes, farms, users are registered on the chain. It plays the central role in achieving decentralised consensus between a user and Node to deploy a certain workload. A contract can be created on the chain that is essentially an agreement between a node and user.
+
+## Twins
+
+A twin is the central Identity object that is used for every entity that lives on the grid. A twin optionally has an IPV6 planetary network address which can be used for communication between twins no matter of the location they are in. A twin is coupled to a private/public keypair on chain. This keypair can hold TFT on TF Chain.
+
+## Farms
+
+A farm must be created before a Node can be booted. Every farms needs to have an unique name and is linked to the Twin that creates the farm. Once a farm is created, a unique ID is generated. This ID can be used to provide to the boot image of a Node.
+
+## Nodes
+
+When a node is booted for the first time, it registers itself on the chain and a unique identity is generated for this Node.
+
+## Node Contract
+
+A node contract is a contract between a user and a Node to deploy a certain workload. The contract is specified as following:
+
+```
+{
+    "contract_id": auto generated,
+    "node_id": unique id of the node,
+    "deployment_data": some additional deployment data
+    "deployment_hash": hash of the deployment definition signed by the user
+    "public_ips": number of public ips to attach to the deployment contract
+}
+```
+
+We don't save the raw workload definition on the chain but only a hash of the definition. After the contract is created, the user must send the raw deployment to the specified node in the contract. He can find where to send this data by looking up the Node's twin and contacting that twin over the planetary network.
+
+## Rent Contract
+
+A rent contract is also a contract between a user and a Node, but instead of being able to reserve a part of the node's capacity, the full capacity is rented. Once a rent contract is created on a Node by a user, only this user can deploy node contracts on this specific node. A discount of 50% is given if a the user wishes to rent the full capacity of a node by creating a rent contract. All node contracts deployed on a node where a user has a rent contract are free of use expect for the public ip's which can be added on a node contract.
+
+## Name Contract
```

Comment:
We didn't document fqdn type?

Reply:
?

Reply:
my bad i confused fqdn proxies with namecontracts